src/dropdown.js

```diff
@@ -37,8 +37,21 @@ export default class extends Controller {
     this.activeClass = this.data.get('activeClass') || null
     this.enteringClass = this.data.get('enteringClass') || null
     this.leavingClass = this.data.get('leavingClass') || null
+
+    if (this.hasButtonTarget) {
+      this.buttonTarget.addEventListener("keydown", this._onMenuButtonKeydown)
+    }
+
+    this.element.setAttribute("aria-haspopup", "true")
```

Comment:
I think I have the right ARIA attributes; but perhaps confirm independently.

Reply:
Looks like it. 👍